torch/_torch_docs.py

```diff
@@ -1903,6 +1903,55 @@ def merge_dicts(*dicts):
     tensor([-1 - 1j, -2 - 2j, 3 + 3j])
 """.format(**common_args))
 
+add_docstr(torch.copysign,
+           r"""
+copysign(input, other, *, out=None) -> Tensor
+
+Returns a new tensor with the magnitude of :attr:`input` and the sign of :attr:`other` elementwise.
+
+.. math::
+    \text{out}_{i} = \begin{cases}
+        -|\text{input}_{i}| & \text{if } \text{other}_{i} < 0 \text{or} \text{other}_{i} == -0.0 \\
+        |\text{input}_{i}| & \text{if } \text{other}_{i} \gt 0 \text{or} \text{other}_{i} == 0.0\\
+    \end{cases}
+""" + r"""
+
+Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`,
+and integer, float inputs. Always promotes integral/boolean/half/float16 types to the float type.
+
+Args:
+    input (Tensor): value with the magnitude of output value.
+    other (Tensor or Number): value with the sign of output value.
```

Comment:
"value with the sign of output value. If the shape of :attr:`input` is different from the shape of :attr:`other`, they must be broadcastable to a common shape (which becomes the shape of the output)." ->
"contains value(s) whose signbit(s) are applied to the magnitudes in :attr:\`input\`."